### PR TITLE
Grammatically upgrated the tech docs

### PR DESCRIPTION
**Pull request by H3c7o4**
Small grammatical updates to the Ivy tech docs.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -80,26 +80,21 @@ Contents
...
+We utilize the "construction" symbol 🚧 to signify that a feature is currently in development, while the "white_check_mark" ✅  denotes features that have already been implemented.
 
-Check out the docs_ for more info,
-and check out our Google Colabs_ for some interactive demos!
+For more information, check out the docs_ and our Google Colabs_ for interactive demos.
 
 🚨 Ivy is still at a relatively early stage of development.
 Expect breaking changes and sharp edges until we release version 1.2.0 in the next few weeks!
 
+It is important to keep in mind that Ivy is still in its early stages of development, and there may be breaking changes and rough edges before the release of version 1.2.0 in the coming weeks. If you want to contribute, please refer to our `contributor guide`_  and `open tasks`_ if you'd like to dive straight in !. 🧑<200d>💻
```
Comment (VedPatwardhan) [inline README.rst:96]: If this line is meant to replace some other lines such as 93-94 and 98-100, the old ones can be removed

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Just a small change, rest looks good to merge 🙂

**Comment by zaeemansari70** [comment]
Take a look at this announcement, I hope this resolves it :slightly_smiling_face: 
https://discordapp.com/channels/799879767196958751/983684937712558080/1080080255387189318

**Comment by zaeemansari70** [comment]
Closing due to inactivity for more than 2 months :slightly_smiling_face:

**Pull request closed without merging**